lib/AmountWithCurrencyField/AmountWithCurrencyField.js

```diff
@@ -8,20 +8,29 @@ import {
 
 import { getAmountWithCurrency } from '../utils';
 
-const AmountWithCurrencyField = ({ stripes, currency, amount = 0 }) => {
+const AmountWithCurrencyField = ({ stripes, currency, amount = 0, showBrackets }) => {
   const usedCurrency = currency || stripes.currency;
+  const withBrackets = showBrackets && amount < 0;
```

Comment:
aren't there no cases when amount is a positive number, but should be displayed with brackets?

Reply:
I'm not sure, the idea of adding this prop to handle negative values. Or what cases do you mean?

Reply:
https://github.com/folio-org/ui-finance/blob/master/src/Transactions/BracketizeTransactionAmount/BracketizeTransactionAmount.js#L11
`if (transaction.fromFundId === fundId && transaction.amount > 0) showBrackets = true;`